Event current/max slots

LCWilliams opened the pull request:
Woo!  First time doing one of these! :p

LCWilliams committed:
Add maximum slot counts to event roles.

LCWilliams committed:
"Fixed" tabs to spaces.

Volumetric-Nanometre commented on src/opsignup.py:
```diff
@@ -306,7 +306,7 @@ async def generic_update_embed(self,obj, message,payload):
                 print(obj.reactions[str(payload.emoji)].members.values())
 
-                memberString = ""
+                memberString = "LIMIT: {obj.reactions[str(payload.emoji)].currentReact} / {obj.reactions[str(payload.emoji)].maxReact}\n"
```
Need to turn this into an F-string

Volumetric-Nanometre commented on src/opsignupclasses.py:
```diff
@@ -147,7 +147,10 @@ async def send_message(self,ctx,date):
+			if self.reactions[reaction].maxReact > 0:
+				embed.add_field( name = f'{self.reactions[reaction].symbol} {self.reactions[reaction].name}', value = f'LIMIT: {self.reactions[reaction].maxReact}\n{self.reactions[reaction].members["perm"]}', inline=True)
+			else:
+				embed.add_field( name = f'{self.reactions[reaction].symbol} {self.reactions[reaction].name}', value = f'{self.reactions[reaction].members["perm"]}', inline=True)
```
Match the formatting of the system with and without a signup.

i.e 0/2 instead of 2

Volumetric-Nanometre replied:
When react max value is changed, the embed needs to be updated.

Volumetric-Nanometre reviewed (changes requested):
Happy to add this after the changes are made in the notes:

1. Convert to f-string
2. Match formatting
3. Force update embed on max-react change

LCWilliams committed:
Fix missing 'f' on formatted string.  Matched curr/max format

LCWilliams committed:
Added updateAllEmbed, updated existing code to make use of changes

Added some QoL changes to react defs:
- Combined name; when using the emoji + title.
- Display name: compiled of CombinedName + min/max.

Edited existing code to make use of combinedName.

New function added and called on update

LCWilliams committed:
Fix overflow math

LCWilliams committed:
Re: add current/max to title.

Can't properly test.  No emojis so it breaks.
